src/main/java/com/polishuchenko/bookstore/repository/order/OrderRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.polishuchenko.bookstore.repository.order;
+
+import com.polishuchenko.bookstore.model.Order;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    Order getOrderByIdAndUserId(Long orderId, Long userId);
```

Comment:
Same here

Reply:
I agree with previous comment, but here I really need userId too, because what if the user wants to view another user's order?